lisa/node.py

```diff
@@ -288,6 +289,8 @@ def execute_async(
         encoding: str = "",
     ) -> Process:
         self.initialize()
+        if isinstance(self, RemoteNode):
```

Comment:
Did you test with local? Maybe local should have the same logic.

Reply:
When hit the same issue with local, I will add the same logic for the local node. But for now, it is not easy to reproduce with local.